src/events/index.js

```diff
@@ -316,6 +317,64 @@ class Events {
       formattedUpdate
     );
   }
+
+  /**
+   * Creates a new event type
+   *
+   * API Endpoint: '/types'
+   * Method: POST
+   *
+   * @param {Object} eventType
+   * @param {string} eventType.name
+   * @param {string} eventType.description
+   * @param {integer} [eventType.level]
+   * @param {string} eventType.clientId
+   * @param {string} eventType.slug
+   * @param {boolean} eventType.isRealtimeEnabled
+   * @param {boolean} eventType.isOngoingEvent
```

Comment:
Can you add a little more info about these keys? There isn't a ton in the regular typedef either